includes/binary_object_upload.form.inc

```diff
@@ -93,27 +93,26 @@ function islandora_binary_object_upload_form_submit(array $form, array &$form_st
   module_load_include('inc', 'islandora', 'includes/utilities');
   $object = islandora_ingest_form_get_object($form_state);
 
-  if (!isset($object['OBJ'])) {
-    $datastream = $object->constructDatastream('OBJ', 'M');
-  }
-  else {
-    $datastream = $object['OBJ'];
-  }
-  $file = file_load($form_state['values']['file']);
-  $datastream->setContentFromFile($file->uri, FALSE);
-  if ($datastream->label != $file->filename) {
-    $datastream->label = $file->filename;
-  }
-  if ($datastream->mimetype != $file->filemime) {
-    $datastream->mimetype = $file->filemime;
-  }
+  if (isset($form_state['values']['file'])) {
```

Comment:
Looking at what's actually passed in the `$form_state`... Looks like it will still be set, will just be zero... Should be able to drop the `isset()`, and just have the condition be the value (similar to the "supply_thumbnail" lower down)... Something like:

```php
if ($form_state['values']['file']) {
[... the rest of the stuff ...]
```

... with the `isset()`, leads to warning attempting to access properties on `$file` (since `file_load(0)` doesn't return anything).